src/collector/main.py

```diff
@@ -110,6 +110,19 @@ def main():
     args.panel_host, panel_metrics, args.panel_modbus_retries,
     args.panel_modbus_retry_wait_time)
 
+  # Add any topics that may not already exist in the database.
+  topics_to_add = []
+  for _, metric_info in panel_metrics.items():
+    topic_name = getattr(metric_info, 'topic_name')
```

Comment:
This loop can be simplified:

```python
for metric in panel_metrics:
    topic_name = metric.topic_name
```

You won't need a variable for topic_name either since it's just an attribute on the metric.

Reply:
I decided to leave in the `topic_name` variable b/c I think it makes the code a bit more readable. Am open to removing it though as well